### ISSUE TITLE
Import models should probable make some verification

### ISSUE THREAD

**Issue by lehors** [lfai/model_openness_tool#9]
The form to enter or edit a model record validates the user input and requires some "mandatory" information to be provided for the submission to succeed.
In contrast, little data is required for a model to be imported into MOT using the `Import models` function. Trying to edit pretty much any of the imported models shows how much is missing.

**Comment by Wassaf001** [comment]
can I work on this?

**Comment by gregcube** [comment]
> can I work on this?

I don't see why not. That said, the CSV import function is rarely used these days—if at all—since we've switched to using pull requests for adding or importing models. But, yeah, feel free to get your hands dirty!

**Comment by lehors** [comment]
I agree with @gregcube. I assume you saw that we have a [schema](https://github.com/lfai/model_openness_tool/blob/main/schema/mof_schema.json). The validation should leverage it. Any help is very much appreciated!

**Comment by Wassaf001** [comment]
ok
